pkg/oc/cli/set/env.go

```diff
@@ -30,9 +28,7 @@ import (
 
 	buildv1 "github.com/openshift/api/build/v1"
 	cmdutil "github.com/openshift/origin/pkg/cmd/util"
-	"github.com/openshift/origin/pkg/oc/util/clientcmd"
 	utilenv "github.com/openshift/origin/pkg/oc/util/env"
-	envresolve "github.com/openshift/origin/pkg/pod/envresolve/internal_version"
```

Comment:
I don't see this package being removed anywhere, and this looks like the last place where it was used. Please remove it.

Reply:
Thanks, done